src/publish.proj

```diff
@@ -8,6 +8,15 @@
     <PublishPattern Condition="'$(PublishPattern)' == ''">$(PackageOutputRoot)**\*.nupkg</PublishPattern>
   </PropertyGroup>
 
+   <PropertyGroup>
+     <PackageDownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)AzureTransfer\$(ConfigurationGroup)</PackageDownloadDirectory>
+     <FinalPublishPattern>$(PackageDownloadDirectory)\**\*.nupkg</FinalPublishPattern>
```

Comment:
Why is this different from PublishPattern above?

Reply:
PublishPattern defines the packages in `bin` that we publish to the intermediate Azure container, FinalPublishPattern defines the packages in `packages/AzureTransfer`, which are the packages we download from that intermediate container (that is, the whole set of packages from all builds)